packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -27,7 +27,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.18.0
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 1.0.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 3.0.0
```

Comment:
The gemspec does not define a version limit, we shouldn't define one in the rpm spec either

Reply:
ack